strapi/strapi
# fix: display & clone relations in components

### What does it do?

<b>FE</b>
* removes the cleaning of data to be cloned when we fetch so we can fetch the component's relations in situ
* omits sending the original id of the entity when cloning on the

<b>BE</b>
* omits `id` if present in the data posted to the clone route

### Why is it needed?

* When cloning an entity all components were not showing the relations from the original entity, in addition these would not be cloned in the BE afterwards.

### How to test it?

* Make an entity with relations inside a component
* clone that entity
* see relations
* click save
* see relations still

### Related issue(s)/PR(s)

* resolves #17220
* resolves CONTENT-1585
